### PR TITLE
meta: switch chalk to ansis

### PR DESCRIPTION
**Pull request by webdiscus**
## Pull Request Checklist

- [ ] Have you added new tests to prevent regressions?
- [ ] If a documentation update is necessary, have you opened a PR to [the documentation repository](https://github.com/sequelize/website/)? 
- [ ] Did you update the typescript typings accordingly (if applicable)?
- [ ] Does the description below contain a link to an existing issue (Closes #[issue]) or a description of the issue you are solving?
- [ ] Does the name of your PR follow [our conventions](https://github.com/sequelize/sequelize/blob/main/CONTRIBUTING.md#6-commit-your-modifications)?

## Description of Changes

Hallo @sdepold,

I'm the author of the [ansis](https://github.com/webdiscus/ansis)  which you already [know](https://github.com/webdiscus/ansis/pull/5).

This PR switch the [chalk](https://github.com/chalk/chalk) to [ansis](https://github.com/webdiscus/ansis) package, because:

- the `sequelize` contains dependencies that already use the `ansis`, e.g. [@oclif/plugin-help](https://github.com/sequelize/sequelize/commit/66fef3592480065add62c886b97f868e702a9fd7)
- ansis is smaller and [faster](https://github.com/webdiscus/ansis#benchmark) than chalk
- ansis supports both `CommonJS` and `ESM`, chalk supports only ESM
- ansis supports `Bun`, `Deno`, `Next.JS` runtimes, chalk has [issues when using Edge runtime](https://github.com/vercel/next.js/issues/50033)
- ansis supports [named import](https://github.com/sequelize/sequelize/compare/main...webdiscus:sequelize:switch-chalk-to-ansis?expand=1#named-import) `import ansis, { red, green, bold, underline } from 'ansis'`, so your can use color names for clean code, e.g.:
   ```js
   this.log(`SQL migration files generated in ${green(migrationPath)}`);
   ```
- other packages already trust ansis and uses `ansis` instead of `chalk`, for example:
   - [facebook/stylex](https://github.com/facebook/stylex/blob/main/packages/cli/package.json)
   - [oclif/core](https://github.com/oclif/core/blob/prerelease/beta/package.json)
   - [grafana/faro-javascript-bundler-plugins](https://github.com/grafana/faro-javascript-bundler-plugins/blob/main/packages/faro-bundlers-shared/package.json)
   - [RobinTail/express-zod-api](https://github.com/RobinTail/express-zod-api/blob/master/package.json)
   - and [thousands others](https://github.com/webdiscus/ansis/network/dependents)
- quick response to issues
- long term support

P.S. It's just right time to use the new `ansis` in the new alpha/beta sequelize :-)

## List of Breaking Changes

no Breaking Changes

### CODE REVIEW

**Comment by fzn0x** [comment]
Looks like this is a better replacement,  we need to looking forward to any alternatives that reduce the repository size.

Screenshot 2024-06-04 145918

Screenshot 2024-06-04 145859

**Review by fzn0x — APPROVED** [review]
LGTM! I think we're good to go!

**Comment by fzn0x** [comment]
I'm not sure why it keep asking for update branch 😄 

Can we need to disable the ability for renovate to merge its PRs?

File: packages/core/src/sequelize-typescript.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ import {
   map,
   splitObject,
 } from '@sequelize/utils';
-import chalk from 'chalk';
+import ansis from 'ansis';
```
Comment (WikiRik) [inline packages/core/src/sequelize-typescript.ts:14]:
```suggestion
import { red, cyan } from 'ansis';
```
And related changes for this

**Review by WikiRik — COMMENTED** [review]
Thanks for opening this PR! I have a minor suggestion, but apart from that it looks good

File: packages/core/src/sequelize-typescript.ts (hunk 1 above)
Comment (webdiscus) [reply packages/core/src/sequelize-typescript.ts]: Hello @WikiRik,

thanks for your suggestion. I have updated the PR using color names.

**Pull request merged**